### ISSUE TITLE
A few improvements for the new installer

### ISSUE THREAD

**Issue by sparky8251** [morph1904/TygerCaddy#65]
Some parts of the installer are missing defaults.
Namely the part where you configure the listening interface & port alongside the dashboard.

The enter key does not behave as expected in all parts of the installer. Sometimes it does nothing, other times it skips sections. Hitting enter should move you through each section.

**Comment by sparky8251** [comment]
Another possible improvement, make it more apparent that DNS verification is optional and what it is really for (one example would be issuing via a port other than 80/443).

**Comment by morph1904** [comment]
I agree, though this should be more comprehensively tackled in the future documentation too. I would like to make much more of this project non-tech user friendly and verbose.
